app/uk/gov/hmrc/crdlcacheadminfrontend/connectors/CRDLConnector.scala

```diff
@@ -144,7 +139,7 @@ class CRDLConnector @Inject() (config: AppConfig, httpClient: HttpClientV2)(usin
       httpClient
         .get(
           urlForCustomsOffices(referenceNumbers = referenceNumbers, countryCodes, roles, activeAt)
-        )(using hcWithInternalAuth)
+        )(using hc)
```

Comment:
In this instance, because we are using something that is already in the implicit scope, we can probably just remove the `using` parameter list entirely.
```suggestion
        )
```